lib/measures/tbd/measure.rb

```diff
@@ -1,8 +1,12 @@
 begin
   # try to load from the gems
+  $STARTING_DIR = Dir.pwd
```

Comment:
This is a workaround to a very bizarre bug I found when testing OpenStudio 3.0.0 and 3.1.0.  I have posted the issue at https://github.com/NREL/OpenStudio/issues/4150

Reply:
That is weird. Thanks for flagging that one.